ReactCommon/react/renderer/attributedstring/conversions.h

```diff
@@ -1033,6 +1087,7 @@ constexpr static MapBuffer::Key TA_KEY_IS_HIGHLIGHTED = 20;
 constexpr static MapBuffer::Key TA_KEY_LAYOUT_DIRECTION = 21;
 constexpr static MapBuffer::Key TA_KEY_ACCESSIBILITY_ROLE = 22;
 constexpr static MapBuffer::Key TA_KEY_LINE_BREAK_STRATEGY = 23;
+constexpr static MapBuffer::Key TA_KEY_ACCESSIBILITY_UNIT = 47;
```

Comment:
hmm... why 47 not 24?

Reply:
@ryancat Thanks a lot. I added the changes with commit https://github.com/fabriziobertoglio1987/react-native/commit/dfc40638ed870587770ec3da286d00661f770378

<details><summary>testing mapping TA_KEY_ACCESSIBILITY_UNIT to 24</summary>
<p>

**ENABLE THE AUDIO**

<video src="https://user-images.githubusercontent.com/24992535/202679113-2658b771-e71f-4e29-8558-cfeffea8b536.mp4" />

</p>
</details>

<details><summary>testing TA_KEY_ACCESSIBILITY_UNIT mapped to different values between Java and CPP (wrong mapping, functionality does not work)</summary>
<p>

**ENABLE THE AUDIO**

<video src="https://user-images.githubusercontent.com/24992535/202679076-1c2b153a-3be0-4fca-a139-9adab7575d02.mp4" />

</p>
</details>